components/meta-packages/history/history

```diff
@@ -896,6 +896,7 @@ library/gnome/print/gnome-print/gnome-print-papi@2.30.0,5.11-2014.0.0.0
 library/gnome/print/gnome-print@2.30.0,5.11-2014.0.0.0
 library/gnu-libiconv@1.14,5.11-2017.0.0.1 noincorporate
 library/gnutls@2.12.24,5.11-2018.0.0.4
+library/ilmbase@2.2.0,5.11-2020.0.1.0
 library/java/cairo-java@1.0.8,5.11-2017.0.0.0
```

Comment:
You'll need to increment the COMPONENT_REVISION here, so make it
library/ilmbase@2.2.0,5.11-2020.0.1.1